release_notes/ocp-4-1-release-notes.adoc

```diff
@@ -395,6 +395,13 @@ SR-IOV hardware on {product-title} nodes, which enables the user to
 attach SR-IOV virtual function (VF) interfaces to Pods in addition to other
 network interfaces.
 
+[id="ocp-4-1-f5"]
+==== F5 support
```

Comment:
Just like 3.11 doc, use `F5 router plug-in support` is better.

Reply:
I agree. I have committed the changes.